evm/src/Transceiver/WormholeTransceiver/WormholeTransceiver.sol

```diff
@@ -271,4 +273,32 @@ contract WormholeTransceiver is
         checkFork(wormholeTransceiver_evmChainId);
         return getWormholePeer(vm.emitterChainId) == vm.emitterAddress;
     }
+
+    function _encodePayload(
```

Comment:
Can we call this `_encodeTransceiverPayload` or something more descriptive instead please?

Reply:
done